docs/api-bindings-cpp.mdx

```diff
@@ -61,6 +61,7 @@ interface PortInfo {
     locationId: string | undefined;
     productId: string | undefined;
     vendorId: string | undefined;
+    friendlyName?: string | undefined;
```

Comment:
Hmm, I was trying to work out if we had set a precedent elsewhere... and the only example I could find was the LowLatency mdoe for Linux.... however upon a closer look, while it is only defined within the Linux specific interface definition, Windows actually returns False.  So I think the PortStatus definition might need to be changed too :-(

Happy to pivot in either direction, but I think it would be good to be consistent in how we handle things across the package...  Can you think of any other platform specific items we might need to consider at the same time?

Reply:
ok, I'll draft an alternative change that includes friendlyname within port info, and lowlatency within get info, but allows them to be undefined